docs/gno-tooling/cli/gnoland.md

```diff
@@ -19,15 +19,10 @@ Starts the Gnoland blockchain node, with accompanying setup.
 | `data-dir`                 | String  | The path to the node's data directory. This is an important folder. The chain may fail to start if this folder is contaminated. (default: `gnoland-data`)                                                                                        |
 | `flag-config-path`         | String  | The flag config file (optional).                                                                                                                                                                                                                 |
 | `genesis`                  | String  | The path to the `genesis.json` file. (default: `genesis.json`)                                                                                                                                                                                   |
-| `genesis-balance-file`     | String  | Initial distribution file. (default: `~/gno/gno.land/genesis/genesis_balances.txt`)                                                                                                                                                              |
```

Comment:
maybe a bit radical here, but if deprecated best remove them from the static doc IMO